packaging/package_info.rst

```diff
@@ -0,0 +1,190 @@
+Package identifiers and information
+====================================
+
+Packages provide two types of information for consumers. The first one would be the package
+identifiers (ID), which is a SHA-1 hash of the package configuration (settings, options, requirements),
+that allows consumers to reuse an existing package without building it again from sources.
+
+The other type of information would be C/C++ build information, as include paths, library names, or
+compile flags
+
+
+Package IDs
+------------
+
+package_id
```

Comment:
And maybe the conanfile.py page should be splitted in several subpages. It's becoming too big.